continuousflex/protocols/protocol_genesis.py

```diff
@@ -0,0 +1,1105 @@
+        form.addSection(label='Inputs')
+        form.addParam('md_program', params.EnumParam, label="MD program", default=PROGRAM_ATDYN,
+                      choices=['ATDYN', 'SPDYN'],
+                      help="SPDYN (Spatial decomposition dynamics) and ATDYN (Atomic decomposition dynamics)"
+                    " share almost the same data structures, subroutines, and modules, but differ in"
+                    " their parallelization schemes. In SPDYN, the spatial decomposition scheme is implemented with new"
+                    " parallel algorithms and GPGPU calculation. In ATDYN, the atomic decomposition scheme"
+                    " is introduced for simplicity. The performance of ATDYN is not comparable to SPDYN due to the"
+                    " simple parallelization scheme but contains new methods and features.", important=True,
+                      expertLevel=params.LEVEL_ADVANCED)
+
+        form.addParam('inputPDB', params.PointerParam,
+                      pointerClass='AtomStruct, SetOfPDBs, SetOfAtomStructs', label="Input PDB (s)",
+                      help='Select the input PDB or set of PDBs.', important=True)
+
+        form.addParam('inputRST', params.FileParam, label="GENESIS Restart File",
+                       help='Restart a previous GENESIS run with a .rst file', default="",expertLevel=params.LEVEL_ADVANCED)
+
+        group = form.addGroup('Forcefield Inputs')
+        group.addParam('forcefield', params.EnumParam, label="Forcefield type", default=0, important=True,
+                      choices=['CHARMM', 'AAGO', 'CAGO'], help="Type of the force field used for energy and force calculation")
+        group.addParam('generateTop', params.BooleanParam, label="Generate topology files ?",
+                      default=False, help="Use the GUI to generate topology files for you (PSF file for CHARMM and TOP file for AAGO/CAGO)."
+                                          " Requires VMD psfgen for CHARMM forcefields "
+                                          " and SMOG2 for GO models. Note that the generated topology files will not include"
+                                          " solvent.")
+        group.addParam('nucleicChoice', params.EnumParam, label="Contains nucleic acids ?", default=0,
+                      choices=['NO', 'RNA', 'DNA'], condition ="generateTop",help="TODo")
+        group.addParam('smog_dir', params.FileParam, label="SMOG2 directory",
+                      help='Path to SMOG2 directory', condition="(forcefield==1 or forcefield==2) and generateTop")
+        group.addParam('inputTOP', params.FileParam, label="GROMACS Topology File",
+                      condition="(forcefield==1 or forcefield==2) and not generateTop",
+                      help='Gromacs ‘top’ file containing information of the system such as atomic masses, charges,'
+                           ' atom connectivities. For details about this format, see the Gromacs web site')
```

Comment:
Can you provide a link of the website of Gromacs? All these files that are required are giving the impression that running the method is only for experts. Can we simplify these files and give them to the user based on some choices? 
One way is to create a wizard that helps the user obtain the files. What do you think?